undefined_syms.txt

```diff
@@ -65,6 +65,201 @@ object_ahg_Skel_0000F0 = 0x060000F0;
 object_bji_Skel_0000F0 = 0x060000F0;
 object_boj_Skel_0000F0 = 0x060000F0;
 
+// segment addresses
```

Comment:
Same as in `variables.h` 
Gameplay keep should be fully decompiled so the proper symbols should be used here.